cmd/release-controller/sync.go

```diff
@@ -514,6 +523,37 @@ func (c *Controller) syncReady(release *Release) error {
 	return nil
 }
 
+func (c *Controller) syncTerminal(release *Release) error {
+	terminalTags := sortedRawReleaseTags(release, releasePhaseAccepted, releasePhaseRejected)
+	if glog.V(5) && len(terminalTags) > 0 {
+		glog.Infof("terminal=%v", tagNames(terminalTags))
+	}
+	for _, releaseTag := range terminalTags {
+		oldVerifyStatus := releaseTag.Annotations[releaseAnnotationVerify]
+		if len(oldVerifyStatus) == 0 {
+			glog.V(5).Infof("%s: Empty verify annotation, skipping", releaseTag.Name)
+			continue
+		}
+		status := make(VerificationStatusMap)
+		if err := json.Unmarshal([]byte(oldVerifyStatus), &status); err != nil {
+			glog.Errorf("Release %s has invalid verification status, ignoring: %v", releaseTag.Name, err)
+			continue
+		}
+		for i := range status {
+			status[i].TransitionTime = nil
+		}
+		newVerifyStatus := toJSONString(status)
+		if  newVerifyStatus == oldVerifyStatus {
+			continue
+		}
+		releasePhase := releaseTag.Annotations[releaseAnnotationPhase]
+		if err := c.ensureReleaseTagPhase(release, []string{releasePhase}, releasePhase, map[string]string{releaseAnnotationVerify: newVerifyStatus}, releaseTag.Name); err != nil {
```

Comment:
hrm, you're updating a bunch of tags in the same stream.  this needs to be "update all of them", then save.  Not update one by one.

Reply:
ok, grouping the tag updates into a single function.